### PR TITLE
made naviation simplier and dynamic

### PR DESCRIPTION
**Pull request by saulflores95**
Hey @LeeSynt4x can you please review the following it handles our dynamic naviation.

### CODE REVIEW

File: src/components/MobileNavbar.tsx [hunk 1]
```diff
@@ -1,23 +1,26 @@
-import { useUser } from "@clerk/nextjs";
+import { UseUserReturn } from "@clerk/nextjs/types/hooks/useUser";
```
Comment (LeeSynt4x) [inline src/components/MobileNavbar.tsx:1]: hey @saulflores95 i think this type is causing issues in our CI/CD tools, please fix this

File: src/components/tables/DashboardTable.tsx [hunk 2]
```diff
@@ -84,7 +84,11 @@ const DashboardTable: React.FC<DashboardTableProps> = ({
                 </thead>
                 <tbody className="divide-y divide-gray-200 bg-white">
                   {[...people, ...people, ...people].map((person) => (
-                    <tr key={person.email}>
+                    <tr
+                      key={`${person.email}-${Math.floor(
```
Comment (LeeSynt4x) [inline src/components/tables/DashboardTable.tsx:88]: Hey @saulflores95 i see you are using a random int for the key, i understand this is static data, lets add a to-do comment to remember to change this to a real ID when we have the data

File: src/components/tables/ResidentialTable.tsx [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+                      className="relative py-3.5 pl-3 pr-4 sm:pr-6"
+                    >
+                      <span className="sr-only">Edit</span>
+                    </th>
+                  </tr>
+                </thead>
+                <tbody className="divide-y divide-gray-200 bg-white">
+                  {[...people, ...people, ...people].map((person) => (
+                    <tr
+                      key={`${person.email}-${Math.floor(
```
Comment (LeeSynt4x) [inline src/components/tables/ResidentialTable.tsx:88]: do the same to-do comment as above.

**Review by LeeSynt4x — CHANGES_REQUESTED** [review]
Just a couple small changes

File: src/components/MobileNavbar.tsx (hunk 1 above)
Comment (saulflores95) [reply src/components/MobileNavbar.tsx:1]: talking a look

File: src/components/tables/DashboardTable.tsx (hunk 2 above)
Comment (saulflores95) [reply src/components/tables/DashboardTable.tsx:88]: I'll add the comment

File: src/components/tables/ResidentialTable.tsx (hunk 3 above)
Comment (saulflores95) [reply src/components/tables/ResidentialTable.tsx:88]: I'll add the comment

**Review by LeeSynt4x — APPROVED** [review]

**Pull request merged**